main/read.c

```diff
@@ -676,61 +671,6 @@ static void fileNewline (void)
 	DebugStatement ( debugPrintf (DEBUG_RAW, "%6ld: ", File.input.lineNumber); )
 }
 
-/*  This function reads a single character from the stream, performing newline
- *  canonicalization.
- */
-static int iFileGetc (void)
-{
-	int	c;
-readnext:
-	c = mio_getc (File.mio);
-
-	/*	If previous character was a newline, then we're starting a line.
-	 */
-	if (File.newLine  &&  c != EOF)
-	{
-		fileNewline ();
-		if (c == '#'  &&  Option.lineDirectives)
-		{
-			if (parseLineDirective ())
-				goto readnext;
-			else
-			{
-				mio_setpos (File.mio, &StartOfLine);
-				c = mio_getc (File.mio);
-			}
-		}
-	}
-
-	if (c == EOF)
-		File.eof = TRUE;
-	else if (c == NEWLINE)
-	{
-		File.newLine = TRUE;
-		mio_getpos (File.mio, &StartOfLine);
-	}
-	else if (c == CRETURN)
-	{
-		/* Turn line breaks into a canonical form. The three commonly
-		 * used forms if line breaks: LF (UNIX/Mac OS X), CR (Mac OS 9),
-		 * and CR-LF (MS-DOS) are converted into a generic newline.
-		 */
-#ifndef macintosh
```

Comment:
Well, haven't found anything but I don't think it's necessary to maintain support for the old platform (no idea whether it actually builds there).